examples/elman/Main.hs

```diff
@@ -0,0 +1,68 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE FunctionalDependencies #-}
+
+module Main where
+
+import Torch.Tensor
+import Torch.DType
+import Torch.TensorFactories
+import Torch.Functions
+import Torch.TensorOptions
+import Torch.Autograd
+
+import Control.Monad.State.Strict
+import Data.List (foldl', scanl', intersperse)
+
+import Parameters
+import RecurrentLayer
+import Utils
```

Comment:
Can we put the cell and optimizer in this file? It wouldn't get too long, but I think it would be much easier to read if we did that.

Reply:
Eh... I think the shorter files would be easier to read? Besides, the example has been cut down to two files anyway.